stdlib/native.rb

```diff
@@ -213,6 +213,10 @@ def Native(obj)
       nil
     elsif native?(obj)
       Native::Object.new(obj)
+    elsif obj.is_a?(Array)
```

Comment:
This is good but should be probably generalized to any bridged class (Array, Numeric, Proc, String, …)

I also have a slight concern about perf.